client/src/components/documents/Document.tsx

```diff
@@ -274,7 +288,7 @@ export default function Document({ user }: DocumentProps) {
                         <div className="flex items-center justify-between mb-2 space-x-2">
                             <i className="text-sm font-light text-tremor-content-strong dark:text-dark-tremor-content-strong">Scale:</i>
                             <i className='text-md font-medium text-tremor-content-strong dark:text-dark-tremor-content-strong'>
-                                {scale.type === ScaleType.ONE_TO_N ? `1: ${scale.scale}` : scale.type}
+                                {scale?.type === ScaleType.ONE_TO_N ? `1: ${scale?.scale}` : scale?.type ?? 'Unknown Scale'}
```

Comment:
scale is mandatory, so i dont't undestrand why you add "?" and why the "Unknown scale"

Reply:
you are right, i found out i had problems with files in my db